coverage/src/BamView-WebComponent.js

```diff
@@ -305,9 +314,6 @@ class BamViewChart extends HTMLElement {
             alert('Invalid chromosome number');
             return false;
         }
-        if (isNaN(start) && isNaN(end)) {
-            return true;
-        }
         if (isNaN(start) || isNaN(end)) {
```

Comment:
As before, it's better not to use isNaN for these sorts of checks.

Reply:
I tested your suggestion, if the start and end input includes some non-numericals, that will break my code. Is "Number.isInteger(start)" better than "isNaN(start)?

Reply:
Yeah actually I like that better